{"name":"formTestcase","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [true] but found [false]","trace":"java.lang.AssertionError: expected [true] but found [false]\n\tat org.testng.Assert.fail(Assert.java:110)\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\n\tat org.testng.Assert.assertTrue(Assert.java:56)\n\tat org.testng.Assert.assertTrue(Assert.java:66)\n\tat com.codecraft.flutter.FormsTest.FromsTest.formTestcase(FromsTest.java:26)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:829)\n\tat org.testng.TestRunner.run(TestRunner.java:602)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:437)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\n\tat org.testng.TestNG.run(TestNG.java:1067)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\n"},"stage":"finished","steps":[{"name":"Selecting the date...dateSelection","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1683445596096,"stop":1683445599147}],"attachments":[{"name":"Failure in method {0}","source":"f0e0d16b-70b8-41bb-923c-81e6274829b4-attachment","type":"image/png"},{"name":"formTestcase failed and screenshot taken!","source":"bf95b71e-7bbc-4c8d-9c8a-26f29f409b82-attachment","type":"text/plain"}],"parameters":[{"name":"environment","value":"IphoneLocal"}],"start":1683445576200,"stop":1683445600589,"uuid":"fc261416-eaf6-478e-9ef5-cd5b2b584a04","historyId":"fac8e9fcc33f5a59e1be6014f8911548","fullName":"com.codecraft.flutter.FormsTest.FromsTest.formTestcase","labels":[{"name":"package","value":"com.codecraft.flutter.FormsTest.FromsTest"},{"name":"testClass","value":"com.codecraft.flutter.FormsTest.FromsTest"},{"name":"testMethod","value":"formTestcase"},{"name":"parentSuite","value":"Flutter Mobile Automation "},{"name":"suite","value":"FLUTTER-5: Regression"},{"name":"subSuite","value":"com.codecraft.flutter.FormsTest.FromsTest"},{"name":"host","value":"Ostans-MacBook-Pro-3.local"},{"name":"thread","value":"93174@Ostans-MacBook-Pro-3.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}